packages/sw-routing/src/lib/router.js

```diff
@@ -122,7 +71,7 @@ class Router {
 
       let responsePromise;
       let matchingRoute;
-      for (let route of (routes || [])) {
+      for (let route of this._routes) {
         if (route.method !== event.request.method) {
```

Comment:
Did we drop support for 'all' like sw-toolbox had?

If so, should we group routes together by method - reducing this loop for each request?